src/language/utils.ts

```diff
@@ -425,3 +425,8 @@ export function getEqualsKind(node: ts.BinaryOperatorToken): EqualsKind | undefi
             return undefined;
     }
 }
+
+export function isStrictNullChecksEnabled(options: ts.CompilerOptions): boolean {
```

Comment:
I think it's okay. We'll probably have to split apart this file some time in the future